### DISCUSSION TITLE
Gagal menjalankan aplikasi Flask UI

### DISCUSSION THREAD

**Discussion by kjanuar** [teamalgoritma/community#831] [Common Error]
Kira-kira apa yang salah ya? 
![image](https://github.com/teamalgoritma/community/assets/151373916/b8a0d71f-6231-4736-9489-3f82e73013f4)

**Comment by arkanayudhistira** [comment]
Dear Pak @kjanuar,

Boleh dipastikan terlebih dahulu pak apakah file app.py ada pada folder "capstone_flaskui" bapak? Kemungkinan terhapus atau sudah berganti nama

**Comment by saskia-dwi-ulfah** [comment]
Dear pak @kjanuar 
Untuk permasalahan tersebut terjadi karena Anda masih belum pindah ke direktori di mana file `app.py` berada. Anda bisa melakukan langkah-langkah berikut:

- Buka file explorer di mana file `app.py` berada.
- Copy path-nya
![image](https://github.com/teamalgoritma/community/assets/73946560/e54696f0-9a68-4820-b586-f7a6119cad7f)
- Buka terminal VS Code, dan `cd` ke path yang sudah di-copy tadi. Contohnya:
```python
cd C:\Users\ASANI\Documents\Algoritma\1. Academy\Xion\DA\capstone_project\algoritma_da_capstone\flask_ui-main
```
- Jalankan `app.py`
```python
python app.py
```
